docs/current_docs/integrations/snippets/argo-workflow.yaml

```diff
@@ -14,15 +14,15 @@ spec:
       sidecars:
         - name: dagger-engine
           # replace with the latest available version of Dagger for your platform
-          image: registry.dagger.io/engine:v0.15.1
+          image: registry.dagger.io/engine:v{{ version }}
           securityContext:
             privileged: true
             capabilities:
               add:
                 - ALL
           readinessProbe:
             exec:
-              command: ["buildctl", "debug", "workers"]
+              command: ["dagger", "core", "version"]
```

Comment:
I didn't understand the reason for this change.

Reply:
I put this change in a separate commit: https://github.com/dagger/dagger/pull/9239/commits/a3818e9f4517fcbea74d9e6298752cf03dceb09c

See https://github.com/dagger/dagger/pull/8219 for context - we should have updated the argo workflow as part of that PR, but it looks like it was missed.